algebras/algebra/src/main/scala/endpoints/algebra/Urls.scala

```diff
@@ -108,10 +111,10 @@ trait Urls {
   }
 
   /** Builds a static path segment */
-  def staticPathSegment(segment: String): Path[Unit]
+  def  staticPathSegment(segment: String): Path[Unit]
 
   /** Builds a path segment carrying an `A` information */
-  def segment[A](implicit s: Segment[A]): Path[A]
+  def segment[A](name: String = "", docs: Documentation = None)(implicit s: Segment[A]): Path[A]
```

Comment:
I would rather have a single way to do things, to not introduce too much confusion to the user. Do you think there is a lot of value in it?

Reg magic value, sure but, what is the benefit over empty string?

Reply:
> I would rather have a single way to do things, to not introduce too much confusion to the user. Do you think there is a lot of value in it?

No, you’re right, let’s start with what you did!

> Reg magic value, sure but, what is the benefit over empty string?

It’s probably highly subjective so it’s ok if you don’t agree with me: I tend to exercise caution with using an empty string to represent some meaningful information. I feel like it’s like using `null` to represent the absence of value, it might make it harder to understand or find issues in our code.

Reply:
I would go with special value if empty string was acceptable value for open api, but I think it isn't, so it makes sense to handle it specially, IMHO.